data/explorations/android_interactions/assets/image/img_20210609_160046_kgk83smxaa_height_150_width_268_micro.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 918.6 515.1" enable-background="new 0 0 918.6 515.1"><style type="text/css">.st0{fill:#3DDC84;}</style><g id="XMLID_1_"><path class="st0" d="M918.6 515.1h-918.6c14.7-155.7 103.7-288.7 235.1-359.9l-76.2-132c-4.3-7.4-1.8-16.8 5.6-21.1s16.8-1.8 21.1 5.6l77.2 133.7c58.9-26.9 125.2-41.9 196.5-41.9s137.6 15 196.5 41.9l77.2-133.7c4.2-7.4 13.7-9.9 21-5.6s9.9 13.7 5.6 21.1l-76.2 132c131.5 71.2 220.5 204.2 235.2 359.9zm-248.5-129c21.3 0 38.6-17.3 38.5-38.5 0-21.2-17.2-38.5-38.5-38.5-21.2 0-38.5 17.2-38.5 38.5 0 21.2 17.2 38.5 38.5 38.5zm-421.7 0c21.3 0 38.6-17.3 38.5-38.5 0-21.2-17.2-38.5-38.5-38.5-21.2 0-38.5 17.2-38.5 38.5 0 21.2 17.2 38.5 38.5 38.5z"/></g></svg>
```

Comment:
This SVG doesn't seem to include any text. Doesn't that make it incompatible with the PNG versions?

Reply:
Sorry I don't get it, also I've updated the images to oppia logo, PTAL

Reply:
Ah, I think my point isn't relevant anymore. Thanks!